lib/index.js

```diff
@@ -157,10 +158,16 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // if `authenticationMethod` is `token-client-registration` then
       // register token client for it
       if(authenticationMethod === 'token-client-registration') {
-        // FIXME: get client ID from cookie
-        const clientId = 'FIXME';
+        // generate a client id
+        const clientId = await generateId({fixedLength: true});
+        // save it to the database
         await brAuthnToken.clients.set(
           {actor: null, email, account, clientId, authenticated: true});
+        // set a cookie in the response
+        const maxAge = cfg.cid.defaults.maxAge;
+        res.setHeader('Set-Cookie', `cid=${clientId}; expires=${maxAge};` +
+          'HttpOnly');
```

Comment:
Is `expires` the right param here for `maxAge`? Also, does `res` have an API for setting cookies or do we have to do it via the headers API?

Reply:
I have updated it to use https://expressjs.com/en/5x/api.html#res.cookie. You are right `expires` is not the right param for maxAge. The value for `expires` should be a date, I removed it . There was a `maxAge` param for maxAge, so used that.